doc-content/enterprise-only/installation/jws-zip-install-proc.adoc

```diff
@@ -10,38 +10,81 @@ To use the installer JAR file to install {KIE_SERVER}, see <<assembly_installing
 ====
 
 .Prerequisites
-* A backed-up {JWS} {JWS_VERSION_LONG} or higher server installation is available. The base directory of the {JWS} installation is referred to as `__JWS_HOME__`. 
+* A backed-up {JWS} {JWS_VERSION_LONG} or higher server installation is available. The base directory of the {JWS} installation is referred to as `_JWS_HOME_`. 
 * Sufficient user permissions to complete the installation are granted.
 
 .Procedure
+ifdef::DM[]
+. Navigate to the https://access.redhat.com/jbossnetwork/restricted/listSoftware.html[Software Downloads] page in the Red Hat Customer Portal (login required), and select the product and version from the drop-down options:
+* *Product:* {PRODUCT_SHORT}
+* *Version:* {PRODUCT_VERSION}
+. Click *Download* next to *{PRODUCT} {PRODUCT_VERSION_LONG} Maven Repository* (`{PRODUCT_FILE}-maven-repository.zip`).
+. Click *Download* next to *{PRODUCT} {PRODUCT_VERSION_LONG} Add Ons* (`{PRODUCT_FILE}-add-ons.zip`).
+. Unzip the `{PRODUCT_FILE}-maven-repository.zip` file. 
+. Unzip the `{PRODUCT_FILE}-add-ons.zip` file.
+. From the unzipped `{PRODUCT_FILE}-add-ons.zip` file, extract the following files:
+* `{PRODUCT_INIT}-{PRODUCT_VERSION}-kie-server-jws.zip`
+* `{PRODUCT_INIT}-{PRODUCT_VERSION}-decision-engine.zip`
+ 
+endif::DM[]
+
+ifdef::PAM[]
 . Download the `{PRODUCT_INIT}-{PRODUCT_VERSION}-kie-server-jws.zip` file:
 .. Navigate to the https://access.redhat.com/jbossnetwork/restricted/listSoftware.html[Software Downloads] page in the Red Hat Customer Portal (login required), and select the product and version from the drop-down options:
 * *Product:* {PRODUCT_SHORT}
 * *Version:* {PRODUCT_VERSION}
-
 .. Click *Download* next to *{PRODUCT} {PRODUCT_VERSION_LONG} Add Ons* (`{PRODUCT_FILE}-add-ons.zip`).
 .. Unzip the `{PRODUCT_FILE}-add-ons.zip` file. The
 `{PRODUCT_INIT}-{PRODUCT_VERSION}-kie-server-jws.zip`
  file is in the unzipped directory.
-. Extract the 
-ifdef::PAM[]
-`{PRODUCT_INIT}-{PRODUCT_VERSION}-kie-server-jws.zip`
+
+. Download the Decision Manager offline Maven repository and Decision Engine:
+.. Navigate to the https://access.redhat.com/jbossnetwork/restricted/listSoftware.html[Software Downloads] page in the Red Hat Customer Portal (login required), and select the product and version from the drop-down options:
+ * *Product:* Decision Manager
```

Comment:
Several occurrences of "Decision Manager" in PAM document.
Also duplicated downloading of add-ons zip.